src/components/LinkItem/LinkItem.type.ts

```diff
@@ -1,28 +1,27 @@
-interface MetaData {
+interface ILinkItem {
   title: string;
   thumbnail: string;
   description: string;
-}
-interface LinkItem extends MetaData {
   linkId: number;
   url: string;
   bookMarkCount: number;
   isRead: boolean;
```

Comment:
```suggestion
  url: string;
  bookMarkCount: number;
  hasRead: boolean;
```
사소하긴한데 isRead보다는 읽은 적 있다라는 의미를 주기위한 hasRead는 어떤가요?